src/GeneralStateTestsFiller/stStaticCall/static_callToDelCallOpCodeCheckFiller.json

```diff
@@ -24,7 +24,7 @@
         "pre" : {
             "1000000000000000000000000000000000000000" : {
                 "balance" : "1000000000000000000",
-		"code" : "Static call success",
+		"code" : "Static call fails because SSTORE is attempted",
```

Comment:
this comment is not an error. static call below MUST return 1 otherwise test fails